### ISSUE TITLE
💡 [REQUEST] - Move process elements up or down by context menu, without drag & drop

### ISSUE THREAD

**Issue by OliverKunc** [dlr-gtlab/gtlab-core#1258]
### Summary

The context menu of a process element should contain the options "Move 1 up" and "Move 1 down", which would re-arrange that process element within the current hierarchy level.

E.g. it is then possible to change the order of root tasks.

It should not be possible to move process elements to another parent or out of their hierarchy.

E.g. consider the following structure:

- Task 1
  - Calc A
  - Calc B
  - Task 1.1
    - Calc C
    - Calc D
  - Calc E
- Task 2

Then relative movements are possible only within each of the following groups:
- Task 1, Task 2
- Calc A, Calc B, Task 1.1, Calc E
- Calc C, Calc D

### Improvements

The currently available drag 'n drop is not user-friendly as it is very sensitive to the position of the mouse.

### Drawbacks

Context menu is enlarged by 2 entries.

### Current workarounds

Succeed with drag 'n drop, delete and re-create the elements, or get used to the order as it is.

**Comment by Patrick-Wehrel** [comment]
As discussed with @jensschmeink , I summarize a bug regarding Tasks here:

Currently, it is not possible to move a Tasks upwards. In the example below, I can't move "Task" between "Calculate Foot and Disk Mass" and "Calculate Disk Stress".

![grafik](https://github.com/dlr-gtlab/gtlab-core/assets/157687096/ec39289e-eed6-4db0-b41c-b509a73b43f0)

It is possible to move "Task" under another Task. When I press "Undo", the task is deleted and any empty task remains.
ble

**Comment by jensschmeink** [comment]
> As discussed with @jensschmeink , I summarize a bug regarding Tasks here:
> 
> Currently, it is not possible to move a Tasks upwards. In the example below, I can't move "Task" between "Calculate Foot and Disk Mass" and "Calculate Disk Stress".
> 
> ![grafik](https://private-user-images.githubusercontent.com/157687096/347326114-ec39289e-eed6-4db0-b41c-b509a73b43f0.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjczNTg4NjYsIm5iZiI6MTcyNzM1ODU2NiwicGF0aCI6Ii8xNTc2ODcwOTYvMzQ3MzI2MTE0LWVjMzkyODllLWVlZDYtNGRiMC1iNDFjLWI1MDlhNzNiNDNmMC5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwOTI2JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDkyNlQxMzQ5MjZaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1jM2JlNjQ4NDg4NDU2ZWRmNjE4Y2VkMWU3ZGI5YzJlNTQ5ZmVhNTI2MDZjNWE3MDg1YmM0ZmI2MDM1YjQ0YWFkJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.BZflx3np-Zj-TLZt__3iu80ay-5NdLT6HzPgteYevIE)
> 
> It is possible to move "Task" under another Task. When I press "Undo", the task is deleted and any empty task remains. ble

@Patrick-Wehrel I think the problem you refer to is fixed in the current version of GTlab. 
I think @OliverKunc thinks about additional options to move elements beyond the current drag'n drop, don't you?

**Comment by OliverKunc** [comment]
Yes, @jensschmeink, I think about moving elements with discrete buttons in the context menu so that even people with limited fine-motor skills such as mine can do this.
